Source/ARTConnection.m

```diff
@@ -303,3 +284,50 @@ + (instancetype)newWithConnectionEvent:(ARTRealtimeConnectionEvent)value {
 }
 
 @end
+
+@implementation ARTConnectionRecoveryKey
+
+- (NSString *)asJson{
+    NSError *error;
+    NSDictionary *object = @{
+        @"msgSerial": @(_msgSerial),
+        @"connectionKey": _connectionKey,
+        @"serials": _serials ? _serials : @{}
+    };
+    
+    NSData *jsonData = [NSJSONSerialization dataWithJSONObject:object
+                                                       options:0
+                                                         error:&error];
+
+    if (!jsonData) {
```

Comment:
This method is already defined as returning `nullable` in interface. I don't think there is a need for a second parameter as recovery key is expected to be null or not-null, so the result will be either. But I am now checking error instead of the jsonData to return nil : 0bdda077a5002392c8efdf928b0fab96c9e0aeec